atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -460,10 +460,20 @@ def add_log_type(output):
                 },
             },
         })
-        annotations = self.workflow.build_result.annotations
-        if annotations and 'digests' in annotations:
-            digests = get_digests_map_from_annotations(annotations['digests'])
```

Comment:
Right, because whenever `koji_prmote` runs it's either in worker build (arrangement >= 2), or in prod build.
In both cases, build_result.annotations is not set.

Reply:
right